tests/components/generic_thermostat/test_climate.py

```diff
@@ -97,7 +97,7 @@ async def test_valid_conf(hass: HomeAssistant) -> None:
 
 
```

Comment:
Please do the typing updates in another PR

Reply:
Fixed.